build/mac/BUILD.gn

```diff
@@ -2,11 +2,11 @@ import("//brave/build/config.gni")
 
 declare_args() {
   mac_signing_identifier = ""
-  mac_signing_keychain = "login"
+  mac_signing_keychain = "keychain"
 }
 
 _packaging_dir = "$root_out_dir/$chrome_product_full_name Packaging"
-keychain_db = getenv("HOME") + "/Library/Keychains/login.${mac_signing_keychain}-db"
+keychain_db = getenv("HOME") + "/Library/Keychains/signing.${mac_signing_keychain}-db"
```

Comment:
> "/Library/Keychains/login.${mac_signing_keychain}-db"

should be imo

> "/Library/Keychains/${mac_signing_keychain}.keychain-db"

The `keychain-db` is always the same and doesn't change, the first part is the keychain name which generally defaults to `login`